### ISSUE TITLE
Increasing # of messages backed up

### ISSUE THREAD

**Issue by brianjking** [bf4/slackup#1]
Can I increase the # of messages past 1,000 by changing the value here?

``` ruby
def write_channel_messages(channel)
    messages = Slack.channels_history(channel: channel["id"], count: "1000")["messages"]
    File.open(backup_filename(channel["name"]), "w")  do |f|
      formatted_messages = format_channel_messages(messages)
      f.write serialize(formatted_messages)
    end
  end
```

**Comment by bf4** [comment]
nope, 'cause that's the api limit.  see https://api.slack.com/methods/channels.history 

However, it can be done with pagination, but I haven't felt the itch to do that yet. I would gladly accept a PR.   The way you can test this locally, is to make the change in your clone of this repo, up the version, rake install, and then use that version wherever you're running it.  Does that help?

I hope this gem has proven useful.  Over time I'd like to evolve it to a better design, tests, etc, but so far it's just a useful hack that 'works for me'  and I thought I'd share.  Did you see how I use it in the README?

**Comment by bf4** [comment]
Are you interested in helping with a PR to paginate?
